include/taskomat/Sequence.h

```diff
@@ -484,7 +484,7 @@ class Sequence
 
 
     /// Check that the given description is valid. If not then throws a task::Error.
-    void check_label(gul14::string_view label);
+    std::string check_label(gul14::string_view& label);
```

Comment:
I guess the documentation should reflect that the function trims whitespace now. Also the function name is no longer really descriptive... maybe it should become `trim_and_check_label()` or something similar.

Reply:
For the docu you must give me more time 😉 .  About the function name: hmm, for this I don't have any opinion but I will rename it to your suggestion.

I also append now the docu for the constructor and `set_label` for trimming whitespaces.